src/ripple/protocol/SField.h

```diff
@@ -266,9 +288,10 @@ class SField
     static int
     compare(const SField& f1, const SField& f2);
 
+    static std::map<int, SField const*> knownCodeToField;
```

Comment:
Instead of making this public consider a function that provides `const&` access.

Reply:
Fixed in https://github.com/XRPLF/rippled/pull/4703/commits/ebe4258ac26d67cb544adfd7c7d693a3d72ce08e